packages/zowe-explorer/CHANGELOG.md

```diff
@@ -59,7 +72,6 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 
 ### New features and enhancements
 
-- Added "Sort Jobs" feature in Jobs tree view: accessible via sort icon or right-clicking on session node. [#2257](https://github.com/zowe/vscode-extension-for-zowe/issues/2257)
```

Comment:
This was removed from the changelog but was added in [v2.12.0](https://github.com/zowe/vscode-extension-for-zowe/releases/tag/v2.12.0)